workflows/nascent.nf

```diff
@@ -14,7 +14,11 @@ include { COVERAGE_GRAPHS } from '../subworkflows/local/coverage_graphs.nf'
 include { TRANSCRIPT_INDENTIFICATION } from '../subworkflows/local/transcript_identification.nf'
 
 include { FASTP } from '../modules/nf-core/fastp/main'
-include { UNTAR as UNTAR_HISAT2_INDEX } from '../modules/nf-core/untar/main'
+include {
+    UNTAR as UNTAR_HISAT2_INDEX
+    UNTAR as UNTAR_STAR_INDEX
+} from '../modules/nf-core/untar/main'
```

Comment:
I'd rather do 2 include statement, it reads nicer IMO

Reply:
Talk to me when someone writes a formatter 😛

Reply:
I like the split includes, but I'll adhere to a style guide once there's a formatter that handles it for me.